godot/src/logic/content_thread.gd

```diff
@@ -31,9 +31,9 @@ func content_processing_count():
 	return _pending_content.size()
 
 
-func _init(id: int, thread: Thread):
-	self.thread = thread
-	self.id = id
+func _init(param_id: int, param_thread: Thread):
+	self.thread = param_thread
+	self.id = param_id
```

Comment:
```suggestion
func _init(param_id: int, param_thread: Thread):
	self.thread = param_thread
	self.id = param_id
```

I don't understand why use `param_` prefix here.. The previous code was ok.

Reply:
GDScript warns you about the shadowing, I use to launch the explorer without GDScript warnings